core/src/com/unciv/models/ruleset/Victory.kt

```diff
@@ -95,6 +99,23 @@ class Milestone(val uniqueDescription: String, private val parentVictory: Victor
         return civsWithCapitals.union(livingCivs)
     }
 
+    /**
+     * Gets the percentage progress of one civilization's countable against another's countable.
+     *
+     * @return The progress percentage (0-100).
+     */
+    @Readonly
+    fun getMoreCountableThanOtherCivPercent(civ: Civilization, otherCiv: Civilization): Float {
```

Comment:
`Milestone::getMoreCountableThanOtherCivPercent` the best place for it? Seems like similar functions are here too.